app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As required for the project, kindly ensure that your application asks for confirmation of email after signing up. You should also have `:confirmable`  enabled in the user model (user.rb file) and then you should enable it in your `db/migrate/20230726004658_add_devise_to_users.rb` file.

Reply:
Hi @tresorsawasawa, thanks for the review it was helpful in understanding the project requirements